### DISCUSSION TITLE
Text render error when set `TransparencyLevelHint`

### DISCUSSION THREAD

**Discussion by Coloryr** [AvaloniaUI/Avalonia#10969] [Q&A]
![$)_}4~4(XTD4CEPEC6A`HT](https://user-images.githubusercontent.com/26276037/230823357-c961c39c-f68c-440d-aa11-f4a4147dae62.png)
![image](https://user-images.githubusercontent.com/26276037/230823420-cb6ee8b8-4af9-4ca9-9002-c3df1494464c.png)

**Comment by Coloryr** [comment]
```
<Window
    x:Class="ColorMC.Gui.UI.Windows.SelfBaseWindow"
    xmlns="https://github.com/avaloniaui"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    xmlns:setting="clr-namespace:ColorMC.Gui.Utils.LaunchSetting"
    xmlns:views="clr-namespace:ColorMC.Gui.UI.Controls"
    xmlns:views1="clr-namespace:ColorMC.Gui.UI.Controls.User"
    Title="Title"
    Width="800"
    Height="450"
    MinWidth="800"
    MinHeight="450"
    d:DesignHeight="450"
    d:DesignWidth="800"
    Background="Transparent"
    ExtendClientAreaChromeHints="NoChrome"
    ExtendClientAreaTitleBarHeightHint="-1"
    ExtendClientAreaToDecorationsHint="True"
    FontFamily="{setting:Font}"
    TransparencyBackgroundFallback="Transparent"
    mc:Ignorable="d">
    <DockPanel>
```

**Comment by Coloryr** [comment]
`TransparencyLevelHint = None`
![image](https://user-images.githubusercontent.com/26276037/230823504-b7e69d9e-78e6-4c0d-b3f8-04559e13a67c.png)

**Comment by Coloryr** [comment]
![](https://user-images.githubusercontent.com/26276037/230824367-8e45c325-d07a-40af-bcd8-885109d008b2.png)
Is `Windows11` bug?

**Comment by Coloryr** [comment]
![488@_Y O69WHI@{511$CAMS](https://user-images.githubusercontent.com/26276037/230825628-ea695d34-4891-4e64-9068-66001667e255.png)

**Comment by jp2masa** [comment]
It's probably this: https://github.com/AvaloniaUI/Avalonia/issues/2464#issuecomment-1500807106.

**Comment by Coloryr** [comment]
fix in windows update
